recipes/flann/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set commit = "40d63fc7db5d1f6dd67364f24e4579aedb57e7b6" %}
+
+{% set tag = environ.get("GIT_DESCRIBE_TAG", "") %}
+{% set number = environ.get("GIT_DESCRIBE_NUMBER", "0") %}
+{% set hash = environ.get("GIT_DESCRIBE_HASH", "0") %}
+{% set tag_ext = ("+" + number + "." + hash) if number != "0" else "" %}
+{% set version = tag + tag_ext %}
+
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: {{ commit }}
```

Comment:
Maybe we should use `git_rev` as we are using a commit. `git_tag` works too, but using it in this way is a bit concerning as this might not be what one would expect. So, I would be a bit concerned that this behavior will go away. Sorry should have fixed this in the PR too.